provision/vvv-init.sh

```diff
@@ -153,10 +153,9 @@ download_wordpress() {
 
 initial_wpconfig() {
   echo " * Setting up wp-config.php"
-  noroot wp core config --dbname="${DB_NAME}" --dbprefix="${DB_PREFIX}" --dbuser=wp --dbpass=wp  --extra-php <<PHP
-define( 'WP_DEBUG', true );
-define( 'SCRIPT_DEBUG', true );
-PHP
+  noroot wp core config --dbname="${DB_NAME}" --dbprefix="${DB_PREFIX}" --dbuser=wp --dbpass=wp
+  noroot wp config set WP_DEBUG true --raw
+  noroot wp config set SCRIPT_DEBUG true --raw
```

Comment:
do the constants need to be wrapped in quotes? Is `--raw` necessary

Reply:
`--raw` is necessary as it prevents the value (`true`) from being treated as a string and wrapped in quotes.